### DISCUSSION TITLE
IntelliJ Ultimate bug. Impossible to use.

### DISCUSSION THREAD

**Discussion by RIP21** [community/community#61017] [Codespaces]
### Select Topic Area

Bug

### Body

I try to start the codespace and it just not connecting despite it being active and cool It did woke it up tho.

Also, it's only possible to launch and normally see codespaces from the screen when all projects are closed. If you try to go through File => Remote Development, it's just an empty list and stuck at `Allow` and `Deny` steps.

I have gh cli installed and I'm logged in there for reference.

**Comment by lshigami** [comment]
yup I also got error with codesnap in IntelliJ

**Comment by gdomingu** [comment]
Hi sorry you are running into issues. Could you provide some more information? 
- Which version of intellij are you using?
- Which version of the Codespaces plugin are you using?
- Can you confirm you have SSH installed on the codespace
- Can you connect to the codespace with gh ssh
- Have you tried connecting with the JetBrains Gateway?
